chain/client/src/sync/epoch.rs

```diff
@@ -483,15 +472,20 @@ impl EpochSync {
             proof.last_epoch.next_next_epoch_info,
         )?;
 
+        let last_epoch_data = &proof.past_epochs[proof.past_epochs.len() - 2];
+        let last_epoch_last_final_block_height = last_epoch_data.last_final_block_header.height();
+        let first_block_info_in_epoch =
+            BlockInfo::from_header(&last_header, last_epoch_last_final_block_height);
```

Comment:
Concern: there is also `from_header_and_endorsements()` method, but `from_header` would copy endorsements from the header. cc @tayfunelmas